chrome-launcher/test/global-mocha-hooks.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const assert = require('assert');
+
+/**
+ * THIS FILE IS A DUPLICATE OF lighthouse-core/test/global-mocha-hooks.js
```

Comment:
nah. don't carry it over.

we'll delete these in a week. they haven't provided value in a long while.